### ISSUE TITLE
cups-ipp-utils should be installed on Ubuntu 16.04

### ISSUE THREAD

**Issue by imphil** [leoarnold/puppet-cups#4]
Hi,

I'm trying to use the module on Ubuntu 16.04 (xenial), which comes with cups 2.1.3. Currently, the module only installs the "cups" package, which does not contain the ipptool binary. So the package should install cups-ipp-utils as well (available since Ubuntu 15.10, see http://packages.ubuntu.com/search?keywords=cups-ipp-utils). For Debian, it looks like the next release will have that package as well (https://packages.debian.org/search?keywords=cups-ipp-utils).

Could you add that to the default list of packages?

Philipp

**Comment by leoarnold** [comment]
Thank you for pointing that out! I'll have a look at it asap.

**Comment by leoarnold** [comment]
This will take a while to find a good conditional to cover Debian and all its derivatives correctly.

In the meantime, I suggest the following workaround for everyone using Ubuntu 15.10 or later 

```Puppet
class { '::cups':
  packages => ['cups', 'cups-ipp-utils'],
}
```

**Comment by imphil** [comment]
Yes, I'm using that workaround at the moment.

In my experience, Ubuntu and Debian require independent treatment. Usually assuming that the currently used naming of packages remains valid for the next release works well (and has less maintenance effort than explicitly specifying the versions). For Ubuntu, I'd do with
~~~
if versioncmp($::lsbdistrelease, '15.10') >= 0 {
// take cups-ipp-utils
~~~

**Comment by leoarnold** [comment]
Fixed in release 1.2.2, published at [Puppet Forge](https://forge.puppet.com/leoarnold/cups/1.2.2) and [GitHub](https://github.com/leoarnold/puppet-cups/releases/tag/1.2.2).
